services/roomService/package-lock.json

```diff
@@ -1,8 +1,10664 @@
 {
```

Comment:
Are these changes to package-lock for the database?  Or just for in general?

Reply:
This is the package lock for the backend, pg is node-postgres which allows us to query the DB
https://node-postgres.com/